MBProgressHUD.m

```diff
@@ -824,6 +858,11 @@ @implementation MBRoundProgressView
 #pragma mark - Lifecycle
 
 - (id)init {
+    // Ensure that this is an accessibility element and set the trait to allow percentage completion to be accessible.
+    self.isAccessibilityElement = YES;
+    self.accessibilityLabel = @"Progress";
```

Comment:
Wrapping with `NSLocalizedString` would make it easier for the host app to provide localizations here. 

```suggestion
    self.accessibilityLabel = NSLocalizedString(@"Progress");
```